app/routes/_auth.signup.tsx

```diff
@@ -0,0 +1,9 @@
+import { SignupForm } from '~/features/signup/singup-form';
+
+export default function Signup() {
+  return (
+    <div className="flex h-screen w-full items-center justify-center px-4">
```

Comment:
If we change `routes/_auth.tsx` to return 
```ts
  return (
    <div className="flex h-screen w-full items-center justify-center px-4">
      <Outlet />
    </div>
  );
  ```
  Then we can avoid duplicating this className

Reply:
I don't want _auth to have any rendering responsibility. it's just there to perform redirect for all auth routes

Reply:
doesn't have to be. depends on the use case. here _auth and _protected are just like route guards